data/search_synonyms.json

```
@@ -32,5 +111,13 @@
     "iv": ["4"],
     "4": ["iv"],
     
-    "Interims": ["Interimshörsäle", "Interims-Audimax"]
+    "Interims": ["Interimshörsäle", "Interims-Audimax"],
+    
+    "Fachschaft": ["FS", "Fachschaftsbüro", "Fachschaftsraum"],
+    "FS": ["Fachschaft", "Fachschaftsbüro", "Fachschaftsraum"],
+    "Fachschaftsbüro": ["Fachschaft", "FS", "Fachschaftsraum"],
+    "Fachschaftsraum": ["Fachschaft", "FS", "Fachschaftsbüro"],
+
+    "AStA": ["SV", "Studentische Vertretung"],
+    "Studentische Vertretung": ["SV", "AStA"],
```

```suggestion
    "AStA": ["SV", "StuVe", "Studentische Vertretung"],
    "Studentische Vertretung": ["SV", "StuVe", "AStA"],
    "StuVe": ["SV", "Studentische Vertretung", "AStA"]
```

StuVe might also be a common enough abbreviation.

(Also, JSON doesn't like trailing commas)